scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -166,13 +166,13 @@
                      'many times. The purpose of this flag is to make it '
                      'possible to write regression tests that are not '
                      'bottlenecked by CNS throughput.')
-flags.DEFINE_string('local_parameter_device', 'gpu',
+flags.DEFINE_enum('local_parameter_device', 'gpu', ('cpu', 'gpu'),
                     'Device to use as parameter server: cpu or gpu. For '
                     'distributed training, it can affect where caching of '
                     'variables happens.')
-flags.DEFINE_string('device', 'gpu',
+flags.DEFINE_enum('device', 'gpu', ('CPU', 'GPU'),
```

Comment:
Let's keep the lowercase versions as well, so the four allowed values would be 'cpu', 'gpu', 'CPU', and 'GPU'. Also have the four values be allowed for 'local_parameter_device'.

Reply:
Done!